google/cloud/security/inventory/transform_util.py

```diff
@@ -39,6 +39,16 @@ def flatten_projects(projects):
     for project in (project for d in projects \
                     for project in d.get('projects', [])):
         project_json = json.dumps(project)
+
+        project_id = project.get('projectId')
+        lifecycle_state = project.get('lifecycleState')
+        parent = project.get('parent', {})
+
+        if lifecycle_state != 'ACTIVE':
```

Comment:
as noted above, the CRM API does not allow filtering on lifecycleState at the moment.